pkgs/development/tools/parsing/tree-sitter/default.nix

```diff
@@ -114,6 +116,19 @@ rustPlatform.buildRustPackage {
     [ which ]
     ++ lib.optionals webUISupport [ emscripten ];
 
+  patches = lib.optionals webUISupport [
+    (fetchpatch2 {
+      name = "stdlib-symbol-fix.patch";
+      url = "https://github.com/tree-sitter/tree-sitter/commit/8ab14a0ee5128a49648e0904b122ecb7e6aa2d3f.patch";
+      hash = "sha256-VE3cB0+MNgj2he4aSUEkxMZO1ZQ3Fzq5uM3emBPE3no=";
+    })
```

Comment:
I believe this commit is included in tree-sitter≥0.21.0. 
`staging` has tree-sitter 0.22.2, so it's going to conflict in the next `staging-next` merge

Reply:
Hot. Yeah it will conflict. What was the CR that landed it? I'll try to retarget if there's work to do, or close if it Just Works.

Reply:
I rebased on staging and removed this patch.